src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMNavigator.java

```diff
@@ -359,6 +449,52 @@ public String getBitbucketServerUrl() {
         return serverUrl;
     }
 
+    @Deprecated
+    @Restricted(DoNotUse.class)
+    @RestrictedSince("2.2.0")
+    @DataBoundSetter
+    public void setBitbucketJenkinsRootUrl(String rootUrl) {
+        if (rootUrl == null || rootUrl.equals("")) {
```

Comment:
use util method to `Util.fixTrimAndEmpty`

Reply:
found a `fixEmptyAndTrim` in Jenkins, seems to be it :)